frontend/src/components/layout/navigation/DashboardSwitcher.tsx

```diff
@@ -18,15 +18,15 @@ export const DashboardSwitcher = () => {
       {/* Email Mask Btn */}
       <Link href="/accounts/profile">
         <a
-          className={`${styles["nav-mask-phone-icon"]} ${
+          className={`${styles["nav-mask-email-icon"]} ${
```

Comment:
This was just an inconsistency — this is the dashboard switcher's email mask link, but it was referring to phone masks.

![image](https://user-images.githubusercontent.com/4251/203376525-80e1c2be-9a2f-4127-840a-604cf5e2fb1a.png)